src/components/tabs/__internal__/tab-title/tab-title.style.js

```diff
@@ -212,28 +201,12 @@ const StyledTitleContent = styled.span`
       }
     `}
 
-  ${({
-    hasSiblings,
-    hasCustomLayout,
-    error,
-    warning,
-    info,
-    size,
-    isTabSelected,
-    position,
-  }) =>
+  ${({ hasSiblings, hasCustomLayout, position }) =>
```

Comment:
Same as above, I think you should also remove these props from <StyledTitleContent ...> in src/components/tabs/__internal__/tab-title/tab-title.component.js